Library/Homebrew/utils.rb

```diff
@@ -121,6 +121,10 @@ def self.git_last_commit
     HOMEBREW_REPOSITORY.cd { `git show -s --format="%cr" HEAD 2>/dev/null`.chuzzle }
   end
 
+  def self.git_last_commit_date
+    HOMEBREW_REPOSITORY.cd { `git show -s --format="%cd" --date=short HEAD 2>/dev/null`.chuzzle }
+  end
```

Comment:
Yeah, it's relative vs. absolute time; I think absolute is probably more useful.